group_vars/development

```diff
@@ -30,6 +32,24 @@ wordpress_sites:
       db_user: example_dbuser
       db_password: example_dbpassword
 
+mail_smtp_server: localhost:1025
+mail_admin: admin@example.dev
+mail_hostname: example.dev
+ssmtp_auth_method: ""
+ssmtp_start_tls: 'no'
+ssmtp_tls: 'no'
+
+ferm_input_list:
+  - type: dport_accept
+    dport: [http, https]
+    filename: nginx_accept
+  - type: dport_accept
+    dport: [8025]
+  - type: dport_limit
+    dport: [ssh]
+    seconds: 300
+    hits: 20
```

Comment:
Let's just disable it in development. This goes a little beyond dev/prod parity.